src/InitOMPasses.hpp

```diff
@@ -5,6 +5,10 @@
 #include "mlir/Pass/Pass.h"
 #include "src/Pass/Passes.hpp"
 
+#ifdef __NNPA__
+#include "src/Accelerators/NNPA/Pass/DLCPasses.hpp"
+#endif
+
 using namespace onnx_mlir;
```

Comment:
Best to avoid using namespace declaration in an header file. (https://stackoverflow.com/questions/5849457/using-namespace-in-c-headers)

Reply:
Should we address the issue of the namespace declaration in a header file with a separate PR? As you see, this PR does not modify Line 12 and is not related.

Reply:
Yes.  I put the comment because I saw the code and I understand is best to land this one and then address the comment in another PR.